### PR TITLE
Uri development

### PR DESCRIPTION
**Pull request by UriBombastic**
Changed spacing on all html files, as well as style.css and script.js

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -20,13 +21,16 @@ <h1>Uri Herszbaum's Portfolio</h1>
...
-      <p>Click here to get a random message:</p>
+      <p>Click here to get a random fun fact:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
       <p> Click <a href="SeniorPictures.html">here</a> to see me peaking in high school</p>
-      <p><a href = "https://www.google.com/">Company I work For For</a></p>
-    <p><a href = "https://www.linkedin.com/in/uri-harding-7335b7183/">LinkedIn</a></p>
-    <p> <a href = "https://github.com/UriBombastic">Github</a></p>
+      <p>
+    <a href = "https://www.linkedin.com/in/uri-harding-7335b7183/"><img src = "images/LinkedinLogo.png" width ="100" height = "100"></a>
```
Comment (cisxingg) [inline portfolio/src/main/webapp/index.html:29]: 2-space indentation:
https://google.github.io/styleguide/htmlcssguide.html#Indentation.
Same below :)

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +33,30 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function randomImage(){
+   // const fs = require('fs');
+    const dir = 'images/senior-pics/';
+
+    //fs.readdir(dir, (err,files) => {
```
Comment (warfa-google) [inline portfolio/src/main/webapp/script.js]: Let's remove this and re-indent the lines below

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +33,30 @@ function addRandomGreeting() {
...
+
+        const imageContainer = document.getElementById('senior-picture-container');
+//remove previous image
+        imageContainer.innerHTML = '';
+        imageContainer.appendChild(imgElement);
+        imageContainer.style.width = "800";//I'm struggling so desperately to constrain this width
+         // });
+}
+
+//tried doing a simple print("hello world") only for Google Chrome to attempt to print my webpage
```
Comment (warfa-google) [inline portfolio/src/main/webapp/script.js:52]: haha. Is this still needed?

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -17,7 +17,14 @@
  */
 function addRandomGreeting() {
   const greetings =
```
Comment (warfa-google) [inline portfolio/src/main/webapp/script.js:19]: change greetings here to something more representative of the content and keep method name and other variables consistent

**Pull request merged**